frontend/src/components/editor/EditorBottomBar.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button, Menu, MenuItem } from "@mui/material";
+import { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { CodeKeyType, selectEditor, setCodeKeyType } from "../../store/editorSlice";
+
+export const BOTTOM_BAR_HEIGHT = 25;
+
+const EditorBottomBar = (props: { width: number | string }) => {
+	const { width } = props;
+	const dispatch = useDispatch();
+	const editorStore = useSelector(selectEditor);
+	const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
+	const open = Boolean(anchorEl);
+
+	const handleOpen = (event: React.MouseEvent<HTMLButtonElement>) => {
+		setAnchorEl(event.currentTarget);
+	};
+
+	const handleClose = () => {
+		setAnchorEl(null);
+	};
+
+	const handleChangeCodeKey = (newKeyCode: CodeKeyType) => {
+		dispatch(setCodeKeyType(newKeyCode));
+		handleClose();
+	};
+
+	return (
+		<div
+			style={{
+				position: "absolute",
+				bottom: 0,
+				left: 0,
+				width,
+				background: "000000",
+				borderTop: "1px solid #ccc",
+				height: BOTTOM_BAR_HEIGHT,
+				display: "flex",
+			}}
+		>
```

Comment:
If we change it to the `Paper` component in MUI, we can support light and dark mode background colors. It also supports the `outlined` variant.

I want to use the colors defined in the MUI design system and plan to manage the color scheme using MUI.